frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -72,7 +73,7 @@ const plugin: Plugin<ConsumedExtensions> = [
       componentProps: {
         name: 'Virtual Machine Templates',
         resource: 'vmtemplates',
-        required: FLAG_KUBEVIRT,
+        required: [FLAG_KUBEVIRT, FLAG_OPENSHIFT],
```

Comment:
please try to use the `FLAGS.OPENSHIFT` constant here. Also it would be better if you moved these changes into your commit